### DISCUSSION TITLE
GSOD Project idea

### DISCUSSION THREAD

**Discussion by CBID2** [pieces-app/opensource#65] [Contribution Ideas]
Currently, there is not an established workflow for internal and external contributions for the documentation. Since Pieces is planning to participate in [Google Season of Docs](https://developers.google.com/season-of-docs/docs/timeline), I think this would be a great project idea.

**Comment by Sophyia7** [comment]
@CBID2 Could you give a more detailed explanation of this idea? Like what do you have in mind?

**Reply by CBID2** [reply]
Hi @Sophyia7. Through my experience contributing to the documentation, there has been a theme of contributors submitting PRs and then ending up having to close it due to the documentation team managing to finish. This can discourage people in the community from contributing. @mason-at-pieces was my inspiration for this idea.

**Reply by seantiz** [reply]
oh yeah, this is very true. I had this experience when translating for a popular JS framework. I have two ideas around Github actions that could avoid a lot of frustration here and increase workflow, but i'll listen to what you want to do.

**Comment by Sophyia7** [comment]
@CBID2 @seantiz Thanks for sharing your thoughts. I will take note of this for GSOD.

**Reply by CBID2** [reply]
@Sophyia7 and @seantiz, according to [GSOD's timeline](https://developers.google.com/season-of-docs/docs/timeline), applications for organizations are due on April 2nd, so we have to move quickly. I highly recommend following [this part of their organization guide](https://developers.google.com/season-of-docs/docs/organization-application-hints#the_application_formhttps://developers.google.com/season-of-docs/docs/organization-application-hints#the_application_form) to help you.

**Reply by Sophyia7** [reply]
Yeah, the team will update the community soon. Thanks for the reminder.
